usefulUtilities/statusIndicator/webApp/app/statusIndicator_webApp.js

```diff
@@ -213,7 +213,7 @@ function getAllEmployees(response) {
             var location = results[i].location ? results[i].location : "Unknown";
 
             if (results[i].status === "busy") {
-                location = "** busy **";
+                location = "**busy**";
```

Comment:
perhaps "Hidden" is more clear than "**busy**" ?

Reply:
changed. agreed